### DISCUSSION TITLE
Flow and redtract are not the same as in the slicer

### DISCUSSION THREAD

**Discussion by bronsn** [mriscoc/Ender3V2S1#943] [Q&A]
Hello

i have a problem with the settings.
If i change the flow in the slicer <1.0 it doesnt change the flow in the firmware itself, on the screen it will always be 100.
No i changed it in the slicer at 1.0 and at the screen there is 105 ??

The same with the retract.
In the Slicer i changed it to 1.5, on the screen at the printer himself  stand: 5.0 ..

Does anyone have an idea where the problem is?

greeting

**Comment by mriscoc** [comment]
Hi, 
You are confusing some slicer settings with printer tune settings. For example, the slicer can change flow by changing directly the extruder movements or by issuing the [flow percentage tune G-code](https://marlinfw.org/docs/gcode/M221.html) `M221`. If the slicer uses the M221 method, you will see changes in the flow value in the screen, but if it uses the first, you don't have any evidence in the screen.

About retract the confusion is similar, there is an article in the Wiki about firmware retract.

**Comment by bronsn** [comment]
Thanks for the fast respond.

I use Prusa Slicer and your explanation make sence but why i see some changes on the display after i set the flow in the slicer at 1.0?
first i set the flow at:
Slicer 0.95 = 100 on the Display
Slicer 0.92 = 100 on the Display

than i set it on
Slicer 1.0 = 105 on the Display ?

This is confusing me )

**Reply by mriscoc** [reply]
Maybe a misconfiguration in the slicer. Look for M221 in the generated G-code file.
